helium/src/main/groovy/com/stanfy/helium/dsl/TypeDsl.groovy

```diff
@@ -41,6 +41,9 @@ class TypeDsl {
     if (args.containsKey("skipUnknownFields")) {
       msg.skipUnknownFields = args.skipUnknownFields
     }
+    if (args.containsKey("parent")) {
```

Comment:
I still prefer "embed" over "parent" :) Because there is not (and should not be) inheritance in helium.